### ISSUE TITLE
Update README and CHANGELOG to emphasize removal of executable

### ISSUE THREAD

**Issue by clintoncwolfe** [inspec/inspec#4011]
### 🎛 Description

In `inspec` 4.3.2, the executable was removed from the gem and is distributed separately in the gem `inspec-bin`.  This isn't clear from the README, and is not loudly called out in the CHANGELOG either, leading to confusion.

Check website for any places, too.

### 🌍 InSpec and Platform Version

4.3.2

### 💁 Possible Solutions

Update docs

Aha! Link: https://chef.aha.io/features/SH-355

**Comment by zenspider** [comment]
We should also add a post-install msg on inspec that the user might want to install inspec-bin as well.

**Comment by darkn3rd** [comment]
Agreed, people outside the company may not know about these changes if they are not communicated.  The website also does not reflect this information either.
